consensus/consensus-types/src/common.rs

```diff
@@ -393,7 +394,13 @@ impl Payload {
         quorum_store_enabled: bool,
     ) -> anyhow::Result<()> {
         match (quorum_store_enabled, self) {
-            (false, Payload::DirectMempool(_)) => Ok(()),
+            (false, Payload::DirectMempool(txns)) => {
+                txns.par_iter().with_min_len(8).try_for_each(|txn| {
```

Comment:
Can we do this in a separate thread pool and also dynamically tune the number of tasks `with_min_len(optimal_min_len(num_txns, 32))`. May be better to refactor the sig verification in `execution_pipeline.rs` file and reuse the same function here?